### DISCUSSION TITLE
Align the discussions between OSPOlogy APAC and CHAOSS APAC

### DISCUSSION THREAD

**Discussion by xiaoya-yaya** [todogroup/ospology#221] [Community Proposals]
## Background
 
Quite a few CHAOSS AP members are of the background of company OSPOers and looking for measurement solutions for their company. CHAOSS has set up the [OSPO working group](https://github.com/chaoss/wg-ospo) and is an OSPO associate of the TODO group.

Currently, we have the AP calls in both communities and similar discussions raised from time to time, however, there are no channels to promote further communications and integrate the output.

## Goal

Align the discussions between CHAOSS APAC and OSPOlogy APAC calls. 

## Proposed Actions

**From 2023, we might**:
- Switch CHAOSS APAC to a Monthly call, at the same time of day as with the OSPOlogy call
- Alternate CHAOSS APAC call and OSPOlogy every two weeks. OSPOlogy APAC calls will focus on how to address context on OSPO sustainability and continuity and CHAOSS APAC calls on how to develop questions and metrics under such context.
- Create events on both community’s meeting calendar
- Promote meetings in Slack channels #chaossaisa, #chapter-apac, #wg-ospometrics-chaoss, etc 
- Record the meeting notes through one uniform document
- Use #wg-ospometrics-chaoss for a common point of communication with the global OSPO WG meetings and APAC CHAOSS and OSPOlogy 

## Expectations
Involve people from both CHAOSS and TODO Group to communicate OSPO-related metrics in the ASIA-PACIFIC region
Output OSPO metrics for OSPO sustainability and continuity in ASIA-PACIFIC orgs

## Deadline

If there is no opposition to this by SC or the community, we will implement these changes starting on Jan 1st, 2023

**Comment by anajsana** [comment]
Thanks @xiaoya-yaya ! FYI: I've pinged the TODO SC so they are in the loop of this and they can provide feedback 😉
